### ISSUE TITLE
Global conditions are missing

### ISSUE THREAD

**Issue by kalagan-75** [paulwetter/DocumentConfigMgrCB#79]
Hi,
as in object, not global conditions are documented at all. It should be wonderful if you can: 
- add a section related to custom global conditions (default are not so useful)
- integrate global conditions used in application requirements
Thanks

**Comment by paulwetter** [comment]
Correct. They are missing. Will have to dig into them and see where they are at. Thanks!

**Comment by paulwetter** [comment]
Probably won't spend time to document all the possibilities for this.  What type do you use?

I'm thinking the following:
* File System (File/Folder)
* Registry Key
* Registry Value
* Script
* WQL query

**Comment by kalagan-75** [comment]
Hi, yes, you are right. Here my considerations:

- Active Directory query: never used, but can be useful (optional)
- Assembly: never used
- File system: yes
- IIS metabase: never used
- Registry key: absolutely yes
- Script: absolutely yes
- SQL query: never used
- WQL query: absolutely yes
- XPath query: never used

But, please, consider also the "expression" condition type, where you can aggregate different global conditions defined as settings type. 
For example (that I use):
1.  I define a global condition based on registry called "computer name"
![computer_name](https://user-images.githubusercontent.com/62688741/108888294-8dc2be00-760b-11eb-82d2-555a4fb26abc.png)
2.  I define a different global condition called "TrendMicro"; this global condition is based on expression type where I use multiple times "computer name" global condition that begins with and a specific string
![trendmicro](https://user-images.githubusercontent.com/62688741/108888305-91eedb80-760b-11eb-865e-ba1b7451d7f3.png)
